docs/source/cleanlab/datalab/guide/issue_type_description.rst

```diff
@@ -141,6 +141,18 @@ To find the underperforming group, Cleanlab clusters the data using the provided
 The underperforming group quality score is equal to `q/r` for examples belonging to the underperforming group, and is equal to 1 for all other examples.
 Advanced users:  If you have pre-computed cluster assignments for each example in the dataset, you can pass them explicitly to :py:meth:`Datalab.find_issues <cleanlab.datalab.datalab.Datalab.find_issues>` using the `cluster_ids` key in the `issue_types` dict argument.  This is useful for tabular datasets where you want to group/slice the data based on a categorical column. An integer encoding of the categorical column can be passed as cluster assignments for finding the underperforming group, based on the data slices you define.
 
+Null Issue
```

Comment:
In the datalab.report(), why do we have this redundant information being printed:

```
Overall dataset quality in terms of this issue: 0.9895
...

Additional Information: 
average_null_score: 0.9895
```

we should not be printing the `average_null_score: 0.9895` line as it's not providing additional value (and is confusing IMO)

Reply:
Good point. `average_null_score` has been removed from the report. `most_common_issue` has been moved to a higher verbosity level. Added a test assertion to verify that no additional info is being printed for the default verbosity level.